includes/wp-admin/class-welcome-fields.php

```diff
@@ -0,0 +1,274 @@
+<?php
+/**
+ * ActivityPub Welcome Fields Class.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\WP_Admin;
+
+use Activitypub\Model\Blog;
+use Activitypub\Model\User;
+use Activitypub\Collection\Actors;
+
+use function Activitypub\get_reply_intent_js;
+use function Activitypub\is_user_disabled;
+
+/**
+ * Class Welcome_Fields.
+ */
+class Welcome_Fields {
+	/**
+	 * Initialize the welcome fields.
+	 */
+	public static function init() {
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_welcome_fields' ) );
+	}
+
+	/**
+	 * Register welcome fields.
+	 */
+	public static function register_welcome_fields() {
+		// Add settings sections.
+		add_settings_section(
+			'activitypub_intro',
+			\__( 'Welcome', 'activitypub' ),
+			array( self::class, 'render_welcome_intro_section' ),
+			'activitypub_welcome'
+		);
+
+		add_settings_section(
+			'activitypub_bookmarklet',
+			\__( 'Bookmarklet', 'activitypub' ),
+			array( self::class, 'render_bookmarklet_section' ),
+			'activitypub_welcome'
+		);
+
+		if ( ! is_user_disabled( Actors::BLOG_USER_ID ) ) :
```

Comment:
No need to change it, it's fine as it is, this is just an FYI: The alternative syntax (if () : endif;) is typically used in template files where PHP and HTML are mixed as it's considered more readable.